Hackney.Shared.HousingSearch/Domain/Accounts/TenureType.cs

```diff
@@ -0,0 +1,20 @@
+namespace Hackney.Shared.HousingSearch.Domain.Accounts
+{
+    public class TenureType
+    {
```

Comment:
could you follow the static factory `Create()` approach and have a private constructor accepting the parameters and setting the fields, and making the fields themselves private?